marketplace.ini

```diff
@@ -79,7 +79,8 @@ tag = %(nuxeo-tag)s
 
 [nuxeo-web-ui]
 mp_to_upload = plugin/web-ui/marketplace/target/*.zip
-
+#depends on marketplace-dam
+other_versions = %(nuxeo-snapshot)s/%(nuxeo-tag)s/%(nuxeo-next_snapshot)s,%(marketplace-dam-snapshot)s/%(marketplace-dam-tag)s/%(marketplace-dam-next_snapshot)s
```

Comment:
In order to work, I think you need to move the whole `[nuxeo-web-ui]` handling after the `[marketplace-dam]` one.

IIRC, the `marketplace-dam-*` will resolve to undefined otherwise...

Reply:
i confirm, the order needs to change for proper var resolution

Reply:
Thanks!